lib/core/datatable/data/datatable-adapter.ts

```diff
@@ -29,6 +29,7 @@ export interface DataTableAdapter {
     getColumns(): Array<DataColumn>;
     setColumns(columns: Array<DataColumn>): void;
     getValue(row: DataRow, col: DataColumn, resolverFn?: (_row: DataRow, _col: DataColumn) => any): any;
+    getType(row: DataRow, col: DataColumn): string;
```

Comment:
I'm sure this is a breaking change, why this is needed compared to just using `col.type` in the HTML like before?